practice.rb

```diff
@@ -0,0 +1,66 @@
+def print_array(a)
+  print "["
+  l = a.length
+  a.each do |item|
+    print "#{item}"
+    if item != a[l-1]
+      print ", "
+    end
+  end
+  print "]"
+end
```

Comment:
LOVE IT! So many people tried to use all ruby's fancy stuff and this is what I was looking for :)